functions/buildActivityBody.js

```diff
@@ -34,9 +36,9 @@ const buildActivityBody = (context) => {
 
   const repositoryUrl = repoUrl ? repoUrl : context.resolvedRemoteGitURl;
   const repoData = context.repoData;
-  const repoName = (repoData && repoData.name) ? repoData.name : null;
-  const repoOrganization = (repoData && repoData.owner) ? repoData.owner : null;
-
+  const repoName = repoData && repoData.name ? repoData.name : null;
+  const repoOrganization = repoData && repoData.owner ? repoData.owner : null;
+  const timestamp = new Date().toISOString();
```

Comment:
Probably would be better to use a different name for this timestamp (timestamp itself is too generic). Maybe something more explicit like `deployTime`